jobs/teleomed_labs_contract.html

```diff
@@ -0,0 +1,47 @@
+---
+title: Backend Python Developer
+company: Teleomed Labs
+url: http://www.teleomed.com
+location: San Diego, CA, USA
+contract: contract
+contact:
+    name: Team
+    email: hello@teleomed.com
+created: !!timestamp '2017-01-13' 
+tags:
+  - django
+  - postgres
+  - drf
+  - security
+---
+
+## Overview
+
+Teleomed Labs is a small development group that specializes in healthcare solutions for vulnerable populations, including low-income seniors, veterans, and people in developing nations. We are currently working on web-based tools to help social workers and care coordinators manage care for vulnerable seniors and identify those at-risk for a variety of life-threatening conditions.
+
+We are looking for an experienced contractor (full-time or part) to join our backend team. You'll be working on things like notifications (SMS/email), activity streams, fine-grained user permissions, search/autocomplete, caching, and chat. Our backend uses the latest Django, hosted on Heroku with a Postgres backend. Our front-end communicates with the backend entirely via JSON API, so no HTML/CSS or UI/UX experience is required.
+
+## Requirements
+
+* Fluency in Python and at least one Python web framework (eg. Django, Flask) 
+* Self-directed with excellent English communication skills
+* Experience working remotely in the past
+* Experience with Git/GitHub and modern development practices (pull requests, code reviews, TDD, etc.)
+* 5+ years of programming experience (internships included), at least 2 years with Python
```

Comment:
Just a suggestion - we find job ads which mandate a specific level of experience tend to put off potentially good candidates. it might be better to suggest that candidates need to be good enough to take a technical leadership role.